src/main/java/net/snowflake/client/jdbc/SnowflakeDatabaseMetaData.java

```diff
@@ -137,6 +122,51 @@ private void raiseSQLExceptionIfConnectionIsClosed() throws SQLException {
     }
   }
 
+  /**
+   * Function to send in-band telemetry data about DatabaseMetadata get API calls and their
+   * associated SHOW commands
+   *
+   * @param resultSet The ResultSet generated from the SHOW command in the function call. Can be of
+   *     type SnowflakeResultSet or SnowflakeDatabaseMetaDataResultSet
+   * @param functionName name of DatabaseMetadata API function call
+   * @param catalog database
+   * @param schema schema
+   * @param generalNamePattern name of table, function, etc
+   * @param specificNamePattern name of table column, function parameter name, etc
+   */
+  private void sendInBandTelemetryMetadataMetrics(
+      ResultSet resultSet,
+      String functionName,
+      String catalog,
+      String schema,
+      String generalNamePattern,
+      String specificNamePattern) {
+    String queryId = "";
+    if (resultSet instanceof SnowflakeResultSet) {
+      try {
+        queryId = resultSet.unwrap(SnowflakeResultSet.class).getQueryID();
+      } catch (SQLException e) {
+        // do nothing here; just don't include the query ID
+      }
+    } else if (resultSet instanceof SnowflakeDatabaseMetaDataResultSet) {
+      try {
+        queryId = resultSet.unwrap(SnowflakeDatabaseMetaDataResultSet.class).getQueryID();
+      } catch (SQLException e) {
+        // do nothing here; just don't include the query ID
+      }
+    }
+    ObjectNode ibValue = mapper.createObjectNode();
+    ibValue.put("type", TelemetryField.METADATA_METRICS.toString());
+    ibValue.put("queryId", queryId);
```

Comment:
please change this to `query_id`

Reply:
Why? We try to make the format consistent with other fields?

Reply:
The current client_telemetry includes this, for example:
```
{
  "query_id": "59648a2e-433c-4272-9d3d-20f1229520ea",
  "type": "client_time_consume_first_result",
  "value": -6
}
```

Reply:
got it!